src/client/tile.cpp

```diff
@@ -2204,6 +2204,45 @@ video::ITexture* TextureSource::getNormalTexture(const std::string &name)
 	return NULL;
 }
 
+namespace {
+	float linear_to_srgb_component(const float v)
```

Comment:
please mark this and the below func as inline (also you don't need the `const float`)

would also be nice to have a comment here where these values come from